### ISSUE TITLE
Ignores options when installed via pip

### ISSUE THREAD

**Issue by hodgesmr** [jpadilla/pyjwt#173]
When I run from source, everything seems to be working properly. However, when installing from pip, I'm seeing unexpected behavior around the `options` param:

```sh
(test) ~/Desktop/test $ pip install PyJWT
Collecting PyJWT
  Using cached PyJWT-1.3.0-py2.py3-none-any.whl
Installing collected packages: PyJWT
Successfully installed PyJWT-1.3.0
(test) ~/Desktop/test $ pip freeze
PyJWT==1.3.0
(test) ~/Desktop/test $ python
Python 2.7.6 (default, Sep  9 2014, 15:04:36)
[GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.39)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import jwt
>>> encoded = jwt.encode({'some': 'payload'}, 'secret', algorithm='HS256')
>>> options = {'require_exp': True}
>>> jwt.decode(encoded, 'secret', algorithms=['HS256'], options=options)
{u'some': u'payload'}
>>> jwt.__version__
'1.3.0'
```

**Comment by hodgesmr** [comment]
I now see this looks like it was added in #166 which isn't part of a current release?

**Comment by jpadilla** [comment]
Just released v1.4.0
